tests/foreman/cli/test_import.py

```diff
@@ -102,3 +108,125 @@ def test_reimport_orgs_default(self):
+                u'fact': u'<%= @ipaddress %>',
+            },
+            {
+                u'name': u'ip_address',
+                u'macro': u'{| rhn.system.net_interface'
+                          u'.ip_address(eth0) |}',
+                u'fact': u'<%= @ipaddress_eth0 %>',
+            },
+            {
+                u'name': u'netmask',
+                u'macro': u'{| rhn.system.net_interface'
+                          u'.netmask(eth0) |}',
+                u'fact': u'<%= @netmask_eth0 %>',
+            },
+            {
+                u'name': u'mac_address',
+                u'macro': u'{| rhn.system.net_interface.'
+                          u'hardware_address(eth0) |}',
+                u'fact': u'<%= @macaddress_eth0 %>',
+            },
+        ]
+        csv_contents = u'\n'.join(
+            u'{0}={1}'.format(i['name'], i['macro']) for i in data
+        )
+        csv_header = [
+            u'org_id', u'channel_id', u'channel', u'channel_type', u'path',
+            u'file_type', u'file_id', u'revision', u'is_binary', u'contents',
+            u'delim_start', u'delim_end', u'username', u'groupname',
+            u'filemode', u'symbolic_link', u'selinux_ctx'
+        ]
+        csv_row = [
+            u'1', u'3', u'config-1', u'normal',
+            u'/etc/sysconfig/rhn/systemid', u'file', u'8', u'1', u'N',
+            csv_contents + u'\n',
+            u'{|', u'|}', u'root', u'root', u'600', u'', u''
+        ]
+
+        file_name = gen_string('alphanumeric', 10) + '.csv'
+        with open('/tmp/' + file_name, 'wb') as csv_file:
+            csv_writer = csv.writer(
+                csv_file, delimiter=',',
+                quotechar='"',
+                quoting=csv.QUOTE_MINIMAL
+            )
+            csv_writer.writerow(csv_header)
+            csv_writer.writerow(csv_row)
+
+        # create a random org that will be mapped to sat5 org with id = 1
+        if bz_bug_is_open(1226981):
+            org_data = {
+                'name': gen_string("alphanumeric"),
+            }
+        else:
+            org_data = {
+                'name': gen_string("utf8"),
+            }
+        org = make_org(org_data)
+        trans_header = [u'sat5', u'sat6', u'delete']
+        trans_row = [u'1', org['id'], u'']
+        trans_file = 'organizations-{0}.csv'.format(
+            gen_string('alphanumeric', 10)
+        )
+        with open('/tmp/' + trans_file, 'wb') as trans_csv:
+            csv_writer = csv.writer(
+                trans_csv, delimiter=',',
+                quotechar='"',
+                quoting=csv.QUOTE_MINIMAL
+            )
+            csv_writer.writerow(trans_header)
+            csv_writer.writerow(trans_row)
+
+        # cleanup the working idrectories and upload the files
+        ssh.command('rm -rf /root/.transition_data /root/puppet_work_dir')
+        self.assertEqual(
+            ssh.command('mkdir -p /root/.transition_data').return_code, 0
```

Comment:
This assertion appears pointless.

    [ichimonji10@beech:tmp]$ mkdir -p foo; echo $?
    0
    [ichimonji10@beech:tmp]$ mkdir -p foo; echo $?
    0